app/Http/Controllers/Admin/MonsterCrudController.php

```diff
@@ -174,29 +199,40 @@ public function setupListOperation()
                 'type'      => 'relationship_count',
                 'name'      => 'categories', // the method that defines the relationship in your Model
                 'entity'    => 'categories', // the method that defines the relationship in your Model
+                'tab'       => 'Relationship',
                 'wrapper'   => [
                     'href' => function ($crud, $column, $entry, $related_key) {
                         return backpack_url('category');
                     },
                 ],
             ],
-            [
-                'name'  => 'video', // The db column name
-                'label' => 'Video', // Table column heading
-                'type'  => 'video',
-            ],
+
         ]);
 
         $this->crud->enableDetailsRow();
         $this->crud->setDetailsRowView('vendor.backpack.crud.details_row.monster');
         $this->crud->enableExportButtons();
         $this->crud->addButtonFromModelFunction('line', 'open_google', 'openGoogle', 'beginning');
 
+        //quickly create a button
+        $this->crud->button('email')->stack('top')->view('crud::buttons.quick')->meta([
+            'access'  => 'Quick Button',
+            'label'   => 'Quick Button',
+            'icon'    => 'la la-fast-forward',
+            'wrapper' => [
+                // 'element' => 'a',
+                'href'   => url('something'),
+                'target' => '_blank',
+                'title'  => 'Create a new email to this user',
+            ],
+        ]);
+
```

Comment:
let's add a button that does something, I don't know: `Go to Backpack Website` or `Go to Documentation` or something similar. But that when clicked something "real" happen.

Reply:
OK Right, I'm targeting it to https://backpackforlaravel.com/docs/crud-buttons#creating-a-quick-button-1
That matches the button title and makes the point.
let me know what you think.